### DISCUSSION TITLE
How to share characters?

### DISCUSSION THREAD

**Discussion by thomasgroendal** [fariapp/fari-app#183] [Q&A] [answered]
I've built some characters for Dresden Files Accelerated here: https://drive.google.com/drive/folders/1QZvaITyiTCUYD_6PcqkDVPO1uRm2tcO1?usp=sharing
I'd love it if I can print them out as paper or PDF so I can share them with prospective players, but for now it appears that the only way I can share these characters is to send them as JSON and coach folks through importing them in Fari. Does anyone have a clever workaround for exporting them as PDFs, etc.?
If they were a bit smaller I'd go for a screenshot, but that's the best I've come up with so far.

**Comment by RPDeshaies** [comment]
Hey!

There's currently no way in the app to export as PDF though you could always do it using your browser' print mode but the result isn't very nice.

![image](https://user-images.githubusercontent.com/6224111/102794522-810abb00-4379-11eb-9186-06a082768283.png)


I'll add an issue in the backlog for the next version because I think it's a pretty nice feature: https://github.com/fariapp/fari/issues/184

For now, I would recommend you coach them on how to import the character in their app :/

**Comment by thomasgroendal** [comment]
Thanks will do!

Thomas Groendal

+1-971-218-0215 (USA Cell)
+81-(0)50-5539-5062 (Japan)
thomasgroendal@hotmail.com
www.hoshudojo.com
________________________________

**Comment by RPDeshaies** [comment, marked as answer]
The next release (which I should be able to release today) will contain a very basic print mode #181 

A better one should be available in the v4.0.0 milestone

I was able to have a pretty good PDF version of your character sheet with the following settings:

![image](https://user-images.githubusercontent.com/6224111/102794479-70f2db80-4379-11eb-8e61-e806ba354c7a.png)

**Reply by RPDeshaies** [reply]
@thomasgroendal new print mode available on fari.app 🚀 

It's basic, but it's better than nothing until a better version is released next year!

**Comment by thomasgroendal** [comment]
This looks like a great start. I’ll have a look.

Thomas Groendal

+1-971-218-0215 (USA Cell)
+81-(0)50-5539-5062 (Japan)
thomasgroendal@hotmail.com
www.hoshudojo.com
________________________________
